irfiacre/Papel
# #171328457 add reset account password

#### What does this PR do?

 This PR adds new and modified files for the credit bank accounts endpoint.

#### Description of Task to be completed? 

- add credit account method to cashier controllers
- add credit route to cashierRoute
- add validations and tests

#### How should this be manually tested?
 - First, you clone the repo on your machine.
 - Open the folder in your text editor,  *preferably   `visual studio code`*
 - Now open your terminal, *for `visual studio code`    just click on    `terminal`* 
 - Now run `npm install`
 - Then  run `npm run dev`,
 - After now open `postman` and run endpoint using POST request *localhost:3000/reset* route; 
- First signup a valid email preferably `Gmail, Yahoo, Hotmail, temp mail`; then fill the email in the request body in this route. Check your mailbox, a reset email would be sent in a few.

#### What are relevant pivotal tracker stories.
https://www.pivotaltracker.com/story/show/171328457

#### screenshots
1. postman
![image](https://user-images.githubusercontent.com/52471122/74805003-968efe80-52ea-11ea-829d-58a238626315.png)

2. Mailbox
![image](https://user-images.githubusercontent.com/52471122/74805019-a7d80b00-52ea-11ea-84e2-be79fca2653f.png)
